sample/kotlin/src/main/kotlin/com/datadog/android/sample/sessionreplay/RadioCheckBoxesFragment.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sample.sessionreplay
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.datadog.android.sample.R
+
+internal class RadioCheckBoxesFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_radio_checkbox_components, container, false)
+    }
+}
```

Comment:
Can be even shorter I think.

```suggestion
internal class RadioCheckBoxesFragment : Fragment(R.layout.fragment_radio_checkbox_components)
```